crud.py

```diff
@@ -0,0 +1,49 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_authors(skip: int, limit: int, db: Session):
+    return db.query(models.Author).offset(skip).limit(limit).all()
+
+
+def get_all_books(db: Session, skip: int = 0, limit: int = 5, author_id: int = None):
+    queryset = db.query(models.Book)
+
+    if author_id:
```

Comment:
The condition `if author_id:` should be `if author_id is not None:` to correctly handle cases where `author_id` is 0, which is a valid integer but would be treated as `False` in a boolean context.